draft-ietf-oauth-transaction-tokens.md

```diff
@@ -303,7 +303,7 @@ Transaction Token (Txn-Token):
 : A signed JWT with a short lifetime, providing immutable information about the user or workload, certain parameters of the call, and specific contextual attributes of the call. The Txn-Token is used to authorize subsequent calls in the call chain.
 
 Authorization Context:
```

Comment:
```suggestion
```

Reply:
the words authorization context appear three other times in the document, before this, with lower case a and c. 
this definition doesn't honestly make sense to me but before tying to figure that out i'd suggest that having it in the terminology isn't even needed